packages/engine/Specs/Scene/Model/ModelImagerySpec.js

```diff
@@ -161,8 +92,37 @@ describe("Scene/Model/ModelImagery", function () {
     expect(modelImagery._imageryConfigurationsModified()).toBeFalse();
   });
 
+  it("considers the show flag as part of the imageryConfigurations", async function () {
+    if (!scene.context.webgl2) {
+      return;
+    }
+
+    const tileset = await loadTilesetWithImagery(scene);
+
+    const root = tileset.root;
+    const content = root.content;
+    const model = content._model;
+    const modelImagery = model._modelImagery;
+    const imageryLayer = tileset.imageryLayers.get(0);
+
+    // Initially, _imageryConfigurationsModified is false (it was just updated)
+    expect(modelImagery._imageryConfigurationsModified()).toBeFalse();
+
+    // For spec: Modify imagery configuration
+    imageryLayer.show = 0.5;
```

Comment:
Shouldn't this be a boolean? I'm hesitant to leave this in the unit tests, as unit test can sometimes serve as an informal way of documenting functionality.

It seems from the comment that you're attempting to trigger a change without hiding the layer. Maybe we can set it to `false` then back to `true`?

Reply:
About the type: "Ooops". (That's what reviews are for...)

About what the test is testing: It is testing that a change in the `show` flag is detected as a change (and this change eventually triggers an update of the draw commands). This is independent of whether it changes to `false` or to `true` (or to 0.5 😬 ).

Setting to `true` could be tested, but that would also only test whether _any_ change of the `show` flag tiggers an update. 

Maybe I misunderstood that point.